server_environment/models/server_env_tech_name_mixin.py

```diff
@@ -40,6 +42,7 @@ class ServerEnvTechNameMixin(models.AbstractModel):
     tech_name = fields.Char(
         help="Unique name for technical purposes. Eg: server env keys.",
         copy=False,
+        required=True,
```

Comment:
- Should not make it required here.
- Make `tech_name ` to be a computed field, readonly=False: get rid of create/write .

Reply:
Thanks, I updated it. Could you have a look it?